app/profile-page.html

```diff
@@ -0,0 +1,143 @@
+      <script src="/bower_components/modernizr/modernizr.js"></script>
+      <!-- endbuild -->
+	  <style>
+		#newPassword{
+			width:50%;
+			float: left;
+		}
+		#confirmPassword{
+			width:50%;
+			float: right;
+		}
+	  </style>
+   </head>
+   <body class="login" ng-app="chataApp">
+      <!--[if lt IE 10]>
+      <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+      <![endif]-->
+      <article class="container" ng-controller="profilePageController+">
+         <section class="page">
+            <div class="holder">
+               <img src="img/logo.png" alt="Chata logo">
+               <h1>Your Profile</h1>
+               <form method="POST" ng-submit="submit()">
+                  <p>Profile Picture (Optional)</p>
+                  <div class="form-group">
+                     <img id="uploadedPicture" src="img/avatar.png" />
+                     <div id="hiddenFileUploader"><input id="upfile" type="file" value="upload" onchange="readURL(this)" /></div>
+                  </div>
+                  <br>
+                  <div class="form-group">
+                     <label class="sr-only" for="username">Username</label>
+                     <input type="text" class="form-control" id="username" placeholder="Username" ng-model="usr_username" >
+                  </div>
+                  <div class="form-group">
+                     <label class="sr-only" for="loginEmail">Email</label>
+                     <input type="email" class="form-control" id="loginEmail" placeholder="Email" ng-model="usr_email" >
+                  </div>
+                  <p class="text-danger" ng-if="error.show"><small>{{error.message}}</small></p>
+                  <fieldset>
+                     <ng-form>
+                        <div class="form-group">
+                           <label class="sr-only" for="loginPassword">Current Password</label>
+                           <input type="password" ng-pattern="/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/" class="form-control" id="loginPassword" placeholder="Current Password" ng-model="usr_password" >
+                           <!--<div ng-show="form.$submitted || form.pass.$touched">
+                              <span ng-show="form.pass.$error.required">Enter your current password.</span>
+                              <span ng-show="form.pass.$error.pattern">Password must have 8 character, 1 letter and 1 number.</span>
+                           </div> -->
+                        </div>
+                        <div class="form-group">
+                           <label class="sr-only" for="newPassword">New Password</label>
+                           <input type="password" ng-pattern="/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/" class="form-control" id="newPassword" placeholder="New Password" ng-model="usr_newPassword" >
```

Comment:
`ng-model="usr_newPassword"` I think this would look nicer as a User Object. So having `user.newPassword` instead etc.